configs/mender_convert_config

```diff
@@ -32,6 +32,8 @@ MENDER_ARTIFACT_COMPRESSION="gzip"
 # this.
 MENDER_ENABLE_SYSTEMD=y
 
+# Set to 'y' for read-only rootfs
+READ_ONLY_ROOTFS=n
```

Comment:
Do we need a new variable here?

If you look at https://github.com/mendersoftware/mender-convert/blob/master/mender-convert-modify#L268-L275, we set the `/etc/fstab` for `/dev/root`.

So if `MENDER_ROOT_PART_FSTAB_OPTS` contains `ro` we could trigger these warnings. Unless it is to big of a hassle to parse in stable way.

Reply:
Makes sense to me, that is a good idea. I was following the suggested fix in the JIRA issue.

@kacf were you thinking otherwise or just an overlook?

Reply:
Hmm, I had not considered that, but I *think* it is a good idea. We are not expecting the read-only property of the system to be defined anywhere else than in `/etc/fstab`, are we? Like setting kernel arguments to `ro` is not enough, the user also needs `fstab`, right?

@drewmoseley, you reported the ticket, does it sound good to you as well?

Reply:
As far as I know that is correct.  I think the kernel command line "ro" may get overridden by whatever is in fstab so I think relying on fstab is the way to go.

Reply:
Cool! Good discussion. I'll rework soon